python/pyhdk/hdk.py

```diff
@@ -0,0 +1,1348 @@
+        delim : str, default: ","
+            Delimiter symbol.
+        header : bool, default: True
+            Use the first CSV file as a header to define column names.
+        skip_rows : int, default: 0
+            Number of rows to skip.
+        fragment_size : int, default: None
+            Number of rows in each table fragment. Total fragments count in a
+            table may affect table processing parallelism level and performance.
+            If not set, then fragment size is chosen automatically.
+        block_size : int, default: None
+            The size of data chunks used by the parser for multi-threaded parsing.
+            Auto-detected by default.
+
+        Returns
+        -------
+        QueryExpr
+            Scan expression referencing created table.
+        """
+        pass
+
+    @not_implemented
+    def import_parquet(self, file_name, table_name=None, fragment_size=None):
+        """
+        Import Parquet file(s) into HDK in-memory storage.
+
+        Parameters
+        ----------
+        file_name : str or list of str
+            Parquet file name or a list of file names. Unix style pathname patterns
+            are allowed to reference multiple files.
+        table_name : str, default: None
+            Destination table name. If not specified, then unique table name is
+            generated and used. If table with specified name already exists,
+            then imported data is appended to the existing table.
+        fragment_size : int, default: None
+            Number of rows in each table fragment. Total fragments count in a
+            table may affect table processing parallelism level and performance.
+            If not set, then fragment size is chosen automatically.
+
+        Returns
+        -------
+        QueryExpr
+            Scan expression referencing created table.
+        """
+        pass
+
+    @not_implemented
+    def import_arrow(self, at, table_name=None, fragment_size=None):
+        """
+        Import Arrow table into HDK in-memory storage.
+
+        Parameters
+        ----------
+        at : pyarrow.Table
+            Arrow table to import.
+        table_name : str, default: None
+            Destination table name. If not specified, then unique table name is
+            generated and used. If table with specified name already exists,
+            then imported data is appended to the existing table.
+        fragment_size : int, default: None
+            Number of rows in each table fragment. Total fragments count in a
+            table may affect table processing parallelism level and performance.
+            If not set, then fragment size is chosen automatically.
+
+        Returns
+        -------
+        QueryExpr
+            Scan expression referencing created table.
+        """
+        pass
+
+    @not_implemented
+    def import_pydict(self, values, table_name=None, fragment_size=None):
```

Comment:
How is this expected to be used?

Reply:
I don't expect it to be used in actual workloads but it can be convenient in small experiments. E.g. in my API tests I use it like the following:
```
hdk = pyhdk.init()
ht = hdk.import_pydict(
    {"a": [1, 2, 3, 4, 5], "b": [5, 4, 3, 2, 1], "x": [1.1, 2.2, 3.3, 4.4, 5.5]}
)
self.check_res(
    ht.proj(a1=ht["a"] // ht["b"], a2=ht["a"] // 2, a3=ht["x"] // 2.0).run(),
    {
        "a1": [0, 0, 1, 2, 5],
        "a2": [0, 1, 1, 2, 2],
        "a3": [0.0, 1.0, 1.0, 2.0, 2.0],
    },
)
```
In fact, it's simply a shorter version for `import_arrow(pyarrow.Table.from_pydict(values))`.